src/controller/java/generated/java/matter/devicecontroller/cluster/clusters/AccountLoginCluster.kt

```diff
@@ -49,75 +82,295 @@ class AccountLoginCluster(
     node: ULong?,
     timedInvokeTimeoutMs: Int
   ) {
-    val commandId = 2L
+    val commandId: UInt = 2u
+    val timeoutMs: Duration = Duration.ofMillis(timedInvokeTimeoutMs.toLong())
+
+    val tlvWriter = TlvWriter()
+    tlvWriter.startStructure(AnonymousTag)
+
+    val TAG_TEMP_ACCOUNT_IDENTIFIER_REQ: Int = 0
+    tlvWriter.put(ContextSpecificTag(TAG_TEMP_ACCOUNT_IDENTIFIER_REQ), tempAccountIdentifier)
+
+    val TAG_SETUP_P_I_N_REQ: Int = 1
+    tlvWriter.put(ContextSpecificTag(TAG_SETUP_P_I_N_REQ), setupPIN)
 
-    // Implementation needs to be added here
+    val TAG_NODE_REQ: Int = 2
+    node?.let { tlvWriter.put(ContextSpecificTag(TAG_NODE_REQ), node) }
+    tlvWriter.endStructure()
+
+    val request: InvokeRequest =
+      InvokeRequest(
+        CommandPath(endpointId, clusterId = CLUSTER_ID, commandId),
+        tlvPayload = tlvWriter.getEncoded(),
+        timedRequest = timeoutMs
+      )
+
+    val response: InvokeResponse = controller.invoke(request)
+    logger.log(Level.INFO, "Invoke command succeeded: ${response}")
   }
 
   suspend fun logout(node: ULong?, timedInvokeTimeoutMs: Int) {
-    val commandId = 3L
+    val commandId: UInt = 3u
+    val timeoutMs: Duration = Duration.ofMillis(timedInvokeTimeoutMs.toLong())
+
+    val tlvWriter = TlvWriter()
+    tlvWriter.startStructure(AnonymousTag)
+
+    val TAG_NODE_REQ: Int = 0
+    node?.let { tlvWriter.put(ContextSpecificTag(TAG_NODE_REQ), node) }
+    tlvWriter.endStructure()
 
-    // Implementation needs to be added here
+    val request: InvokeRequest =
+      InvokeRequest(
+        CommandPath(endpointId, clusterId = CLUSTER_ID, commandId),
+        tlvPayload = tlvWriter.getEncoded(),
+        timedRequest = timeoutMs
+      )
+
+    val response: InvokeResponse = controller.invoke(request)
+    logger.log(Level.INFO, "Invoke command succeeded: ${response}")
```

Comment:
maybe severity of this should be lower (debug?)

Reply:
update to Level.FINE